### PR TITLE
refactor/message: refactor message and several logs

### PR DESCRIPTION
**Pull request by Verbalist**

### CODE REVIEW

File: core/service/block.ts [hunk 1]
```diff
@@ -510,7 +510,7 @@ class BlockService {
 
         if (errors.length) {
             this.pushInPool(removedTransactions);
-            logger.error(`[Process][newReceiveBlock] ${JSON.stringify(errors)}`);
+            logger.error(`[Process][newReceiveBlock] blockId: ${block.id} errors: ${JSON.stringify(errors)}`);
```
Comment (axelsheva) [inline core/service/block.ts]: [Service][Block][Process][receiveBlock]

File: api/controller/round/index.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { RPC } from 'api/utils/decorators';
 import SocketMiddleware from 'api/middleware/socket';
 import { API_ACTION_TYPES } from 'shared/driver/socket/codes';
-import {Message2} from 'shared/model/message';
+import {Message} from 'shared/model/message';
```
Comment (macaqueee) [inline api/controller/round/index.ts]: formatter

File: api/controller/round/index.ts [hunk 3]
```diff
@@ -14,18 +14,18 @@ class RoundController {
...
     @RPC(API_ACTION_TYPES.GET_ROUNDS)
-    getRounds(message: Message2<Pagination>, socket: any) {
+    getRounds(message: Message<Pagination>, socket: any) {
         SocketMiddleware.emitToCore<Pagination>(message, socket);
     }
 
     @RPC(API_ACTION_TYPES.GET_ROUND)
     @validate()
-    getRound(message: Message2<{height: number}>, socket: any) {
+    getRound(message: Message<{height: number}>, socket: any) {
```
Comment (macaqueee) [inline api/controller/round/index.ts]: formatter

File: api/controller/referredUsers/index.ts [hunk 4]
```diff
@@ -12,7 +13,7 @@ export class ReferredUsersController {
 
     @RPC(API_ACTION_TYPES.GET_REFERRED_USERS)
     @validate()
-    getReferredUsers(message: Message, socket: any) {
+    getReferredUsers(message: Message<ResponseEntity<{ address: string, level: number }>>, socket: any) {
```
Comment (macaqueee) [inline api/controller/referredUsers/index.ts]: ResponseEntity?

**Pull request merged**